shuffle-plugin/src/main/scala/com/nvidia/spark/rapids/shuffle/ucx/UCX.scala

```diff
@@ -183,6 +183,7 @@ class UCX(transport: UCXShuffleTransport, executor: BlockManagerId, rapidsConf:
       worker = context.newWorker(workerParams)
       logInfo(s"UCX Worker created")
       if (rapidsConf.shuffleUcxUseSockaddr) {
+        val secureRandom = new SecureRandom()
```

Comment:
Nit: Why not move this within the one block that needs it?  If the port is specified in the conf then this will still construct an unused instance.

Reply:
Fixed!